apps/dialtone-documentation/docs/.vuepress/theme/client.js

```diff
@@ -172,127 +181,131 @@ async function importDocumentation (app) {
-      // '@dialpad/dialtone-tokens/themes/126-light.js': (await import('@dialpad/dialtone-tokens/themes/126-light')),
-      // '@dialpad/dialtone-tokens/themes/126-dark.js': (await import('@dialpad/dialtone-tokens/themes/126-dark')),
-      // '@dialpad/dialtone-tokens/themes/127-light.js': (await import('@dialpad/dialtone-tokens/themes/127-light')),
-      // '@dialpad/dialtone-tokens/themes/127-dark.js': (await import('@dialpad/dialtone-tokens/themes/127-dark')),
-      // '@dialpad/dialtone-tokens/themes/128-light.js': (await import('@dialpad/dialtone-tokens/themes/128-light')),
-      // '@dialpad/dialtone-tokens/themes/128-dark.js': (await import('@dialpad/dialtone-tokens/themes/128-dark')),
-      // '@dialpad/dialtone-tokens/themes/129-light.js': (await import('@dialpad/dialtone-tokens/themes/129-light')),
-      // '@dialpad/dialtone-tokens/themes/129-dark.js': (await import('@dialpad/dialtone-tokens/themes/129-dark')),
-      // '@dialpad/dialtone-tokens/themes/130-light.js': (await import('@dialpad/dialtone-tokens/themes/130-light')),
-      // '@dialpad/dialtone-tokens/themes/130-dark.js': (await import('@dialpad/dialtone-tokens/themes/130-dark')),
-      // '@dialpad/dialtone-tokens/themes/131-light.js': (await import('@dialpad/dialtone-tokens/themes/131-light')),
-      // '@dialpad/dialtone-tokens/themes/131-dark.js': (await import('@dialpad/dialtone-tokens/themes/131-dark')),
-      // '@dialpad/dialtone-tokens/themes/132-light.js': (await import('@dialpad/dialtone-tokens/themes/132-light')),
-      // '@dialpad/dialtone-tokens/themes/132-dark.js': (await import('@dialpad/dialtone-tokens/themes/132-dark')),
-      // '@dialpad/dialtone-tokens/themes/133-light.js': (await import('@dialpad/dialtone-tokens/themes/133-light')),
-      // '@dialpad/dialtone-tokens/themes/133-dark.js': (await import('@dialpad/dialtone-tokens/themes/133-dark')),
-      // '@dialpad/dialtone-tokens/themes/134-light.js': (await import('@dialpad/dialtone-tokens/themes/134-light')),
-      // '@dialpad/dialtone-tokens/themes/134-dark.js': (await import('@dialpad/dialtone-tokens/themes/134-dark')),
-      // '@dialpad/dialtone-tokens/themes/135-light.js': (await import('@dialpad/dialtone-tokens/themes/135-light')),
-      // '@dialpad/dialtone-tokens/themes/135-dark.js': (await import('@dialpad/dialtone-tokens/themes/135-dark')),
-      // '@dialpad/dialtone-tokens/themes/136-light.js': (await import('@dialpad/dialtone-tokens/themes/136-light')),
-      // '@dialpad/dialtone-tokens/themes/136-dark.js': (await import('@dialpad/dialtone-tokens/themes/136-dark')),
-      '@dialpad/dialtone-tokens/themes/137-light.js': (await import('@dialpad/dialtone-tokens/themes/137-light')),
-      '@dialpad/dialtone-tokens/themes/137-dark.js': (await import('@dialpad/dialtone-tokens/themes/137-dark')),
+    // Import all themes explicitly (Vite doesn't like dynamic imports)
+    const themeModules = await Promise.all([
```

Comment:
Could probably import these dynamically with a loop since there are so many, but not important for now I suppose.